static/src/js/components/Facets/Facets.jsx

```diff
@@ -47,6 +49,10 @@ const Facets = (props) => {
     featuresFacet.children.push({
       applied: featureFacets.availableInEarthdataCloud,
       title: 'Available in Earthdata Cloud',
+      iconProps: {
+        icon: CloudFill,
+        label: 'a cloud icon'
```

Comment:
Depending on how you decide to define the label, this could be changed to `ariaLabel` or `"aria-label".

Also, in these labels, we should probably be sure to use proper capitalization, i.e. "A cloud icon".